usl_pipeline/cloud_functions/main_test.py

```diff
@@ -636,9 +636,9 @@ def test_compute_custom_wps_variables_wind():
     processed_ds = main._compute_custom_wps_variables(ds)
 
     # Check that processed dataset contains newly computed variables
-    assert all(var in processed_ds.keys() for var in ["WSPD10", "WDIR10"])
+    assert all(var in processed_ds.keys() for var in ["WSPD", "WDIR_SIN", "WDIR_COS"])
```

Comment:
should we have some tests for the computed values as well?

Reply:
@MadhaviJain90 is this something you can help validate?

Reply:
can you use some simpler wind vectors as test inputs? like, one that runs exactly east?

Reply:
The only thing that is concerning to me is that all values of sin and cos are always negative? The scaling was from 1 to -1. So we should be seeing some positive values too

Reply:
although not knowing exactly what they represent, I'd assume you'll get simpler values to reason about with unit vectors like [1, 0, 0] & such

Reply:
Thanks for the suggestion. updated the test. feels right but can't confirm my trig is bad
@MadhaviJain90 take a look at the expected values in test assertions. if it looks good, i'll merge!

Reply:
Perf!